src/app/public/src/modules/open-on-click-directive/open-on-click.directive.ts

```diff
@@ -0,0 +1,26 @@
+import { Directive, HostListener, Input, Renderer2, ElementRef } from '@angular/core';
+
+import { HelpWidgetService } from '../shared';
+
+@Directive({
+  selector: '[bbHelpOpenOnClick]'
+})
+export class BBHelpOpenOnClickDirective {
+  @Input()
+  private bbHelpOpenOnClick: string;
+
+  constructor (
+    private widgetService: HelpWidgetService,
+    private elementRef: ElementRef,
+    private renderer: Renderer2) {
+      this.renderer.setStyle(this.elementRef.nativeElement, 'cursor', 'pointer');
+  }
+
+  @HostListener('click', ['$event'])
+  public onClick(event: MouseEvent): boolean {
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
What's the reasoning behind prevent/stopping this event from bubbling up? Seems that this would cause adverse behavior if it was placed inside of another component that also had a click event?

Reply:
I thought if it was an anchor tag it would try to navigate/refresh the page, but it seems to work fine without it. good find 👍